docs/src/components/demo/chosung-includes-demo.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+import { chosungIncludes } from 'es-hangul';
+
+export function ChosungIncludesDemo() {
+  const [searchWord, setSearchWord] = useState<string>('홍길동');
+  const [userInput, setUserInput] = useState<string>('ㅎㄱㄷ');
+
+  const result = chosungIncludes(searchWord, userInput);
+
+  return (
+    <div className="bg-white rounded-lg shadow-md p-6 max-w-md mx-auto my-8">
```

Comment:
https://github.com/toss/es-hangul/pull/101#discussion_r1613103807

아직 다크모드 제거가 결정이 나지 않은 만큼 대응이 필요하다고 생각해요. 다크모드에 맞는 색과 조화가 있으면 해요

```suggestion
    <div className="bg-white rounded-lg shadow-md p-6 max-w-md mx-auto my-8">
```

<img width="886" alt="image" src="https://github.com/toss/es-hangul/assets/61593290/6e0432ce-11c7-4198-a1c2-c7ddbeb01765">